model/clusters_mgmt/v1/control_plane_upgrade_policy_type.model

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an upgrade policy that can be set for a cluster.
+class ControlPlaneUpgradePolicy {
+	// Cluster ID this upgrade policy for control plane is defined for.
+	ClusterID String
+
+	// Schedule cron expression that defines automatic upgrade scheduling.
+	Schedule String
+
+	// Schedule type can be either "manual" (single execution) or "automatic" (re-occurring).
+	ScheduleType String
+
+	// Upgrade type specify the type of the upgrade. Can be "OSD" or "CVE".
```

Comment:
Can be only "ControlPlane" at this time.

Reply:
hmm, considering this endpoint is specific for this. Does it make sense to have this attribute at all?

Reply:
The purpose of this is to later support other types, like CVE upgrades. I suggest to keep it at this time, to allow this kind of flexibility.